components/script/dom/closeevent.rs

```diff
@@ -45,9 +45,7 @@ impl CloseEvent {
         let ev = reflect_dom_object(event, global, CloseEventBinding::Wrap);
         {
             let event = ev.upcast::<Event>();
-            event.init_event(type_,
-                             bubbles == EventBubbles::Bubbles,
-                             cancelable == EventCancelable::Cancelable);
```

Comment:
I think one of the reasons why it was preferred to have an enum struct instead of a tuple Boolean struct was because this is much more readable. `bubbles.0` doesn't give the reader much information about what the code is actually doing.